did-sdk/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
need to figure out how to move this all to repo root so we can stop duplicating it in every package

Reply:
Are you referring to cascading
https://eslint.org/docs/user-guide/configuring/configuration-files#cascading-and-hierarchy

Reply:
yes but i think it runs into some hiccups in monorepos cause it stops looking after a package.json

but i haven't really looked into this yet. not enough friction to really warrant it 😅